src/OAM.md

```diff
@@ -122,4 +122,6 @@ This can be exploited to only hide parts of an object behind the background
 ([video demonstration](https://youtu.be/B8sJGgCVvnk)).
 A similar behaviour [can be seen on the NES](https://forums.nesdev.org/viewtopic.php?f=10&t=16861).
 
+In CGB mode interaction between BG and OBJ is declared in more than one register, [See background and window to obj priority](Tile_Maps.md#bg-to-obj-priority-in-cgb-mode)
```

Comment:
```suggestion
**In CGB Mode**, BG vs. OBJ priority is declared in more than one register, [please see this page](Tile_Maps.md#bg-to-obj-priority-in-cgb-mode) for more details.
```

Maybe the tip box should be renamed "priority over BG", too?

Reply:
I'm not really sure, most of this tip is about the internal priority between objects